src/test/java/jUnit/JsonParserTest.java

```diff
@@ -1,4 +1,83 @@
 package jUnit;
 
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.junit.jupiter.params.provider.ValueSource;
+import parser.JsonParser;
+import parser.NoSuchFileException;
+import shop.Cart;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
 public class JsonParserTest {
+
+    Cart initialCart;
+    JsonParser parser;
+
+    List<String> fileNameList = Stream.of(
+            "vadim-cart",
+            "vadim1-cart",
+            "vadim2-cart")
+            .collect(Collectors.toList());
+
+    @BeforeAll
+    void setUp() {
+        parser = new JsonParser();
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {0, 1, 2})
+    void writeToFileTest(int i) {
+        initialCart = new Cart(fileNameList.get(i));
+        parser.writeToFile(initialCart);
+        Cart actualCart = parser.readFromFile(new File("src/main/resources/" + fileNameList.get(i) + ".json"));
+        assertAll(() -> assertEquals(initialCart.getCartName(), actualCart.getCartName()),
+                () -> assertEquals(initialCart.getTotalPrice(), actualCart.getTotalPrice()));
+    }
+
+    @Test
+    public void readFromFileTest() {
```

Comment:
readFromFile test might be improved - you may want to create virtual item, real item, add both to cart and write that info to file. Then only verify total price with tax and cart name is correct.

This change is required as eugen-cart might get changed and that, in turn, will break down the test. To avoid that let's create it inside the test case and then verify if write-read works as expected. Plus you will avoid using magic numbers and strings inside test case